apps/auth0-rules/src/__tests__/connect-user.test.ts

```diff
@@ -96,11 +97,13 @@ describe('Auth0 Rule - Connect User', () => {
     nock.cleanAll();
   });
 
-  it('should callback with same user + context if receives no invitation_code', () => {
-    const cb: jest.MockedFunction<Parameters<typeof connectUser>[2]> =
-      jest.fn();
+  afterEach(() => {
+    cb.mockClear();
```

Comment:
I saw this in apps/auth0-rules/src/__tests__/add-user-metadata.test.ts also but should we be using mockClear or mockReset?

I guess they will both set it back to jest.fn()?

Reply:
here we probably could but in the other one we had other mocks too which we didnt want to clear between tests, i think its better to be more specific here